src/main/resources/init_db.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE book (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR(255),
+    price DECIMAL(10, 2)
+)
```

Comment:
```suggestion
)

```

Reply:
not fixed
always ensure you have an empty line at the end of each file
![image](https://github.com/mate-academy/jv-jdbc-intro/assets/49767049/8ed72633-c8c4-4a25-807b-f942d8c35ae3)